docs/basic_construction.md

```diff
@@ -13,7 +13,7 @@ This process is guided by a random oracle, which evaluates the validity of each
 At each step, the random oracle performs a check, and only tuples that satisfy this check are extended further. 
 This ensures that the proof construction process proceeds efficiently by discarding invalid tuples early.
 
-The proof construction employs *Depth-First Search (DFS)* over a conceptual tree, where each path represents a potential tuple. 
+The proof construction employs *Depth-First Search (DFS)* over a conceptual tree, where each node represents a set element, and each leaf represents a potential proof. To guarantee the generation of a proof, we iterate the DFS algorithm on several trees, each marked by a counter. An Alba proof can thus be seen as the Merkle proof of a polytree where each underlying tree's root is our counter.
```

Comment:
I think the comparison to merkle proof is very misleading.

Reply:
Both are Merkle path from the root to a leave though? The verification method of course is different, in Merkle proof you need to have the "associated" nodes, hash them together and check the final output is the root, while in our case we do not need associated data but just perform oracle checks, still some hash but with extra comparisons, recursively.

Reply:
Sorry I made mistake, I meant a tree path.